graphql/resolvers.go

```diff
@@ -435,7 +435,8 @@ func (r *taskResolver) CanOverrideDependencies(ctx context.Context, at *restMode
 		RequiredLevel: evergreen.TasksAdmin.Value,
 		Resource:      *at.ProjectId,
 	}
-	if len(at.DependsOn) > 0 && currentUser.HasPermission(requiredPermission) {
+	if len(at.DependsOn) > 0 && (currentUser.HasPermission(requiredPermission) ||
+		utility.StringSliceContains(evergreen.PatchRequesters, utility.FromStringPtr(at.Requester))) {
```

Comment:
The ticket does note that this shouldn't include commit queue patches, which PatchRequesters does include, so we may just want to explicitly check the correct patch requesters.

Reply:
done